capsules/src/process_console.rs

```diff
@@ -170,6 +176,49 @@ impl Default for WriterState {
     }
 }
 
+#[derive(Copy, Clone)]
+pub struct Command {
+    buf: [u8; COMMAND_BUF_LEN],
+    len: usize,
+}
+impl Command {
+    pub fn new() -> Command {
+        Command {
+            buf: [0; COMMAND_BUF_LEN],
+            len: 0,
+        }
+    }
```

Comment:
What about implementing the [`Default`](https://doc.rust-lang.org/std/default/trait.Default.html) trait?

Reply:
Does implementing it have any downsides ? Thought more about performance. If not, the `same_bytes` function can be replaced with implementing `PartialEq`, maybe even `Eq`, as I see it.

Reply:
I don't see any downside, and this is the Rust way of doing it.